Dockerfile

```
@@ -3,7 +3,7 @@
 # * oc login api.ci.openshift.org ... # command from the web page
 # * oc registry login
 # * skopeo copy docker://registry.svc.ci.openshift.org/ocp/builder:golang-1.15 docker:/quay.io/redhat-developer/servicebinding-operator:builder-golang-1.15
-FROM quay.io/redhat-developer/servicebinding-operator:builder-golang-1.16 AS builder
+FROM quay.io/bmuthuka/go-toolset:1.17.12-3 AS builder
```

This needs to be changed.
```
docker pull registry.redhat.io/rhel8/go-toolset:1.17.12-3
docker tag registry.redhat.io/rhel8/go-toolset:1.17.12-3 quay.io/redhat-developer/servicebinding-operator:1.17.12-3
docker push  quay.io/redhat-developer/servicebinding-operator:1.17.12-3
```
```suggestion
FROM quay.io/redhat-developer/servicebinding-operator:1.17.12-3 AS builder
```

---

Since we are using `distroless` as the base image, I changed the builder to `golang`. This should work for upstream releases and testing.